x-pack/plugins/stack_connectors/public/connector_types/security/index.ts

```diff
@@ -4,3 +4,5 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+
```

Comment:
This connector is only for Security by now, I saw the directory was already created and I guessed it was intended for this kind of connector. If it is not the case I can move everything to `stack`.

Reply:
@jonathan-buttner Sorry for the late reply! Yes, I added this as a placeholder since I was told there would be security specific connector types but it sounds like everything will be moving to back to one location eventually anyway :)